src/controllers/module/event/eventController.ts

```diff
@@ -566,7 +566,7 @@ export class EventController implements FormController {
 	public updateBooking() {
 		return async (req: Request, res: Response) => {
 			const bookings = await this.learnerRecord.getEventBookings(req.params.eventId)
-			const bookingId = req.params.bookingId
+			const bookingId = +req.params.bookingId
```

Comment:
`+` is a quick way for converting string to integer? I can see from corresponding test, `bookingId` is of string type